ec/ecresource/deploymentresource/schema.go

```diff
@@ -56,6 +56,13 @@ func NewSchema() map[string]*schema.Schema {
 			Sensitive: true,
 		},
 
+		// APM secret_token
+		"apm_secret_token": {
```

Comment:
Not against this change but wondering why are we moving it up one level?

Reply:
it's to make it like the `elasticsearch_username` and `elasticsearch_password ` fields (same file L49-55)